hikari/impl/entity_factory.py

```diff
@@ -1748,6 +1748,16 @@ def _deserialize_command_option(self, payload: data_binding.JSONObject) -> comma
         if raw_channel_types := payload.get("channel_types"):
             channel_types = [channel_models.ChannelType(channel_type) for channel_type in raw_channel_types]
 
+        name_localizations: typing.Mapping[str, str] = {}
+        if raw_name_localizations := payload.get("name_localizations"):
+            name_localizations = {locales.Locale(k): raw_name_localizations[k] for k in raw_name_localizations}
+
+        description_localizations: typing.Mapping[str, str] = {}
+        if raw_description_localizations := payload.get("description_localizations"):
+            description_localizations = {
+                locales.Locale(k): raw_description_localizations[k] for k in raw_description_localizations
```

Comment:
For these lines of code it would be more fluent and quicker to iterate over both the key and value by doing `for key, value in raw_description_localizations.items()`

Reply:
Also since you're building a mapping here you probably want to do
```py
If ...:

else:
    description_localizations = {}
 ```
 to avoid throwing away a dict each time

Reply:
Just quick thing, doing `.items()` might read better but its definitely slower unfortunately

Reply:
> For these lines of code it would be more fluent and quicker to iterate over both the key and value by doing `for key, value in raw_description_localizations.items()`

That's what I originally did, I changed it after [dav's request](https://github.com/hikari-py/hikari/pull/1141/files/565cdf170626c8896442c998fa1c17fbb75fa948#r857899974).

Reply:
Isn't this exact pattern of putting the default value before the if statement used in many other places already? It is certainly easier for the reader and for formatting.

Reply:
>  Isn't this exact pattern of putting the default value before the if statement used in many other places already? It is certainly easier for the reader and for formatting.
That's only done for singletons

Reply:
> Just quick thing, doing .items() might read better but its definitely slower unfortunately

It does just read better and is more standard so I would usually prefer .items()

Reply:
So, do I change it?

Reply:
The difference is minuscule in the grand scheme of things.